### PR TITLE
Add test for ViewPlainPtr

### PR DESCRIPTION
**Pull request by BenjaminW3**
This is a preparation for #655 and does not yet fix or change anything.

### CODE REVIEW

**Comment by BenjaminW3** [comment]
push!

File: test/unit/mem/view/src/ViewPlainPtrTest.cpp [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+namespace alpaka
+{
+namespace test
+{
+namespace mem
+{
+namespace view
+{
+    //#############################################################################
+    //! 1D: sizeof(TIdx) * (5)
```
Comment (ax3l) [inline test/unit/mem/view/src/ViewPlainPtrTest.cpp]: outdated comments?

File: test/unit/mem/view/src/ViewPlainPtrTest.cpp [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+    //! 2D: sizeof(TIdx) * (5, 4)
+    //! 3D: sizeof(TIdx) * (5, 4, 3)
+    //! 4D: sizeof(TIdx) * (5, 4, 3, 2)
+    // We have to be careful with the extents used.
+    // When Idx is a 8 bit signed integer and Dim is 4, the extent is extremely limited.
```
Comment (ax3l) [inline test/unit/mem/view/src/ViewPlainPtrTest.cpp]: it's `size_t` now (and the name is `Tidx` not `Idx`?)

File: test/unit/mem/view/src/ViewPlainPtrTest.cpp [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+    //#############################################################################
+    template<
+        std::size_t Tidx>
```
Comment (ax3l) [inline test/unit/mem/view/src/ViewPlainPtrTest.cpp]: Shouldn't this be spelled `TIdx`?

File: test/unit/mem/view/src/ViewPlainPtrTest.cpp (hunk 1 above)
Comment (BenjaminW3) [reply test/unit/mem/view/src/ViewPlainPtrTest.cpp]: You are right, this is outdated but not only here but in all such places. I will create a follow up PR to fix this everywhere.

File: test/unit/mem/view/src/ViewPlainPtrTest.cpp (hunk 2 above)
Comment (BenjaminW3) [reply test/unit/mem/view/src/ViewPlainPtrTest.cpp]: Here as well...

File: test/unit/mem/view/src/ViewPlainPtrTest.cpp (hunk 1 above)
Comment (BenjaminW3) [reply test/unit/mem/view/src/ViewPlainPtrTest.cpp]: I will move this whole helper struct into a common place for all tests.

**Pull request merged**